.github/workflows/schedule-dependencies.yml

```diff
@@ -30,9 +30,36 @@ jobs:
     - name: Install dependencies
       run: |
         uv pip install wheel --system
-        uv pip install ${{ matrix.pre-release-dependencies }} scikit-lego --system
+        uv pip install ${{ matrix.pre-release-dependencies }} --system -e ".[test-dep]"
         uv pip freeze
     - name: Test with pytest
+      run: pytest -n auto --disable-warnings --cov=sklego -m "not cvxpy and not formulaic and not umap"
```

Comment:
As far as I know, there is no easy way to ask only for those tests without a "registered" mark.
I tried to achieve that dynamically via [pytest_collection_modifyitems](https://docs.pytest.org/en/latest/reference/reference.html#pytest.hookspec.pytest_collection_modifyitems), but it's a bit of a headache as `@pytest.mark.parametrize` and any other pytest marker count as a mark in the items

Reply:
As-is seems perfectly fine :)